broker/src/main/java/io/camunda/zeebe/broker/system/configuration/backup/S3BackupStoreConfig.java

```diff
@@ -20,6 +20,7 @@ public class S3BackupStoreConfig implements ConfigurationEntry {
   private String secretKey;
   private Duration apiCallTimeout = Duration.ofSeconds(180);
   private boolean forcePathStyleAccess = false;
+  private boolean enableCompression = false;
```

Comment:
:question:  Do you think we will have to support other compression algorithms in future? If yes, for future extendibility, instead of a boolean flag, it would be better to specify the compression algorithm here. See `ZEEBE_BROKER_CLUSTER_MESSAGECOMPRESSION` as an example. 
```
ZEEBE_BROKER_DATA_BACKUP_S3_COMPRESSION = none | zstd
```
This way we can add new compression algorithms without changing the configuration format. What do you think?

NOTE: It is not clear to me if there is value in letting users choose the compression algo. So I'm ok to leave it as it is.